deploy/helm/sumologic/templates/_helpers.tpl

```diff
@@ -1262,3 +1310,38 @@ Example Usage:
 {{ toYaml (get (get .Values .Provider) "serviceLabels") }}
 {{- end }}
 {{- end -}}
+
+{{/*
+Endpoint used by otelagent exporter.
+
+Example Usage:
+{{- $otelcolService := include "traces.otelagent.exporter.endpoint" . }}
+
+*/}}
+{{- define "traces.otelagent.exporter.endpoint" -}}
+{{- if (eq .Values.otelgateway.enabled true) }}
+{{- printf "%s.%s" ( include "sumologic.metadata.name.otelgateway.service" . ) .Release.Namespace }}
+{{- else }}
+{{- printf "%s.%s" ( include "sumologic.metadata.name.otelcol.service" . ) .Release.Namespace }}
+{{- end }}
+{{- end -}}
+
+{{/*
+Endpoint used by otelgateway otlp exporter.
+
+Example Usage:
+'{{ include "traces.otelgateway.exporter.loadbalancing.endpoint" . }}'
```

Comment:
This is not the same template name as below: `traces.otelgateway.exporter.endpoint`